ejabberd.te

```diff
@@ -0,0 +1,57 @@
+policy_module(ejabberd,0.0)
+
```

Comment:
Needs to be ordered and commented per the [Style Guide](https://github.com/TresysTechnology/refpolicy/wiki/StyleGuide), including the header comment blocks like in other modules.